core/src/main/java/org/dst/core/operatorImpl/DstListImpl.java

```diff
@@ -1,12 +1,16 @@
 package org.dst.core.operatorImpl;
 
-import org.dst.core.exception.NotImplementException;
 import org.dst.core.operatorset.DstList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.HashMap;
 import java.util.List;
 
 public class DstListImpl implements DstList {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(DstListImpl.class.getName());
```

Comment:
This can be simply to `LoggerFactory.getLogger(DstListImpl.class);`

Reply:
This is what I copied from the test class，I'll fix both of them later。